litmus-portal/graphql-server/server.go

```diff
@@ -26,7 +26,7 @@ const defaultPort = "8080"
 var err error
 
 func init() {
-	if os.Getenv("JWT_SECRET") == "" || os.Getenv("SERVICE_ADDRESS") == "" || os.Getenv("SUBSCRIBER_IMAGE") == "" || os.Getenv("DEPLOYER_IMAGE") == "" {
+	if os.Getenv("JWT_SECRET") == "" || os.Getenv("SUBSCRIBER_IMAGE") == "" || os.Getenv("DEPLOYER_IMAGE") == "" {
```

Comment:
other env checks can also be added. cc: @gdsoumya

Reply:
There are so many env vars now doing it like this is not efficient

Reply:
Anyways I put those envs :)